Makefile

```diff
@@ -31,11 +31,13 @@ IMAGE_REPO :=quay.io/microshift/microshift
 IMAGE_REPO_AIO :=quay.io/microshift/microshift-aio
 OUTPUT_DIR :=_output
 RPM_BUILD_DIR :=packaging/rpm/_rpmbuild
+ISO_DIR :=scripts/image-builder/_builds
 CROSS_BUILD_BINDIR :=$(OUTPUT_DIR)/bin
 FROM_SOURCE :=false
 CTR_CMD :=$(or $(shell which podman 2>/dev/null), $(shell which docker 2>/dev/null))
 ARCH :=$(shell uname -m |sed -e "s/x86_64/amd64/" |sed -e "s/aarch64/arm64/")
 IPTABLES :=nft
+PULLSECRET :=~/pullsecret
```

Comment:
We're using pull secrets (or better: registry credentials) in other places, too, e.g. the rebase script.
Would it make sense to align on a common location for these, e.g. `$HOME/.docker/config.json` that `oc adm`, podman, etc. default to?

Reply:
Good point.

Reply:
I commented below, asking to align the location with the one documented in [Install MicroShift on RHEL for Edge](https://github.com/openshift/microshift/blob/main/docs/rhel4edge_iso.md#prerequisites).

I think we should change the docs if it does elsewhere. I used ~/pull-secret.txt location - keeping the file name downloaded from console.redhat.com

Reply:
After an offline discussion with @fzdarsky, we recommend to use `$HOME/.pull-secret.json` location for the secret files. This is not specific to docker/container and also a hidden file.

@jogeo, can you also make the changes in the docs?
```
$ grep -r pull-secret.txt docs
docs/rhel4edge_iso.md:Download the OpenShift pull secret from the https://console.redhat.com/openshift/downloads#tool-pull-secret page and save it into the `~microshift/pull-secret.txt` file. 
docs/rhel4edge_iso.md:./scripts/image-builder/build.sh -pull_secret_file ~/pull-secret.txt
docs/rhel4edge_iso.md:./packaging/rpm/make-microshift-images-rpm.sh rpm ~/pull-secret.txt x86_64:amd64 rhel-8-x86_64
docs/rhel4edge_iso.md:./scripts/image-builder/build.sh -pull_secret_file ~/pull-secret.txt -custom_rpms ~/microshift/packaging/rpm/paack-result/microshift-containers-4.10.18-1.x86_64.rpm
```